data/json/monsters/nether.json

```diff
@@ -110,8 +109,7 @@
     "melee_cut": 2,
     "harvest": "zombie",
     "special_attacks": [ [ "FEAR_PARALYZE", 0 ], { "type": "bite", "cooldown": 5 } ],
-    "death_drops": "default_zombie_clothes",
-    "death_function": [ "NORMAL" ],
+    "death_drops": "amigara_drops",
```

Comment:
Why is the blood sacrifice being changed? It just has a `NORMAL` death func.

Reply:
i seem to have made a find-and-replace error. i'll get it reverted